### PR TITLE
End to end AgriSeafoodOpService

### PR DESCRIPTION
**Pull request by jujaga**
# Description

This PR adds the agriSeafoodOpScreeningService and updates all relevant stores, components, constants and services to allow for full end-to-end communication.
- Note that the agriSeafoodOpScreeningService will likely be refactored to be more common in the very near future.

[SHOWCASE-1201](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1201)

## Types of changes

Bug fix (non-breaking change which fixes an issue)
New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-vmvfjv-tools.pathfinder.gov.bc.ca/dashboard?id=common-forms-toolkit-pr-88

File: app/frontend/src/components/agriseafoodopscreening/admin/SubmissionsTable.vue [hunk 1]
```diff
@@ -48,7 +48,7 @@
 
 <script>
 import GeneratePdfButton from '@/components/common/GeneratePdfButton.vue';
-import minesOperatorScreeningService from '@/services/minesOperatorScreeningService';
+import agriSeafoodOpScreeningService from '@/services/agriSeafoodOpScreeningService';
 
 export default {
   name: 'MinesSubmissionsTable',
```
Comment (TimCsaky) [inline app/frontend/src/components/agriseafoodopscreening/admin/SubmissionsTable.vue:54]: i think this needs changing at some point

Comment (jujaga) [reply app/frontend/src/components/agriseafoodopscreening/admin/SubmissionsTable.vue]: Fixed

File: app/frontend/src/components/agriseafoodopscreening/admin/AdminReviewSubmission.vue [hunk 2]
```diff
@@ -41,21 +41,21 @@
...
-import Step4 from '@/components/minesoperatorscreening/Step4.vue';
-import Step5 from '@/components/minesoperatorscreening/Step5.vue';
+import Step1 from '@/components/agriseafoodopscreening/Step1.vue';
+import Step2 from '@/components/agriseafoodopscreening/Step2.vue';
+import Step3 from '@/components/agriseafoodopscreening/Step3.vue';
+import Step4 from '@/components/agriseafoodopscreening/Step4.vue';
 
 export default {
-  name: 'ReviewSubmission',
+  name: 'AdminReviewSubmission',
```
Comment (loneil) [inline app/frontend/src/components/agriseafoodopscreening/admin/AdminReviewSubmission.vue:50]: Probably needs an agriculture-specific name for now or would be a duplicate?

Comment (jujaga) [reply app/frontend/src/components/agriseafoodopscreening/admin/AdminReviewSubmission.vue:50]: We can get by for now having a generic name as the DOM by how we organize won't have collisions - we only have one type of this component per view. But yes, either we refactor long-term so this is actually only one common component, or we make the name more specific based on directory context too.

Let's revisit this when we have a chance to refactor...

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-vmvfjv-tools.pathfinder.gov.bc.ca/dashboard?id=common-forms-toolkit-pr-88

**Pull request merged**